package.json

```diff
@@ -35,13 +35,21 @@
   },
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "^4.14.0",
     "hof-bootstrap": "^6.0.0",
+    "hof-controllers": "^4.1.2",
+    "hof-form-controller": "^1.0.0",
     "hof-frontend-toolkit": "^1.0.0",
+    "hof-model": "^2.1.0",
+    "hof-template-mixins": "^2.0.0",
     "hof-transpiler": "0.1.0",
+    "hogan.js": "^3.0.2",
     "homeoffice-countries": "^0.1.0",
     "jquery": "^3.1.1",
+    "lodash": "^4.16.3",
     "node-fetch": "^1.6.3",
     "node-sass": "^3.7.0",
+    "so-acceptance": "^4.2.2",
```

Comment:
Good spot, I've updated. 

So for my learning the dependencies section should be those that are needed for production and devdependencies are just for development purposes.  To be aware of this when installing stuff DevDependencies I guess are mainly dependencies that are used for testing, linting, grunting and all that jazz